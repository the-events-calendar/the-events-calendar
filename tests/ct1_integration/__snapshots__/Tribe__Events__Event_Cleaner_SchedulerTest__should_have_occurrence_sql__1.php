<?php return '
			SELECT post_id
			FROM wp_posts AS t1
			INNER JOIN wp_postmeta AS t2 ON t1.ID = t2.post_id
			WHERE
				t1.post_type = %s
				AND t2.meta_key = \'_EventEndDate\'
				AND t2.meta_value <= DATE_SUB( CURRENT_TIMESTAMP(), INTERVAL %d %3s )
				AND t2.meta_value != 0
				AND t2.meta_value != \'\'
				AND t2.meta_value IS NOT NULL
				AND t1.post_parent = 0
				AND t1.ID NOT IN ( 
			SELECT DISTINCT post_parent
			FROM wp_posts
			WHERE
				post_type= \'tribe_events\'
				AND post_parent <> 0
		 )
			LIMIT %d
		';
