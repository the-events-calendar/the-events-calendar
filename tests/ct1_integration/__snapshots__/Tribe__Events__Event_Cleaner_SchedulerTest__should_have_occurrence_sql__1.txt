SELECT test_tec_occurrences.post_id
				FROM test_posts
			    	INNER JOIN test_tec_occurrences ON test_posts.ID = test_tec_occurrences.post_id
				WHERE test_posts.post_type = %s
					AND test_tec_occurrences.end_date_utc <= DATE_SUB( CURRENT_TIMESTAMP(), INTERVAL %d %3s )
					AND test_posts.post_status != 'trash'
				GROUP BY test_tec_occurrences.post_id
				HAVING COUNT(*) = 1
				ORDER BY test_tec_occurrences.start_date_utc ASC, test_tec_occurrences.end_date_utc ASC
				LIMIT %d